src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -77,4 +77,51 @@ public function committeeAutocompleteAction(Request $request)
 
         return new JsonResponse($result ?? []);
     }
+
+    /**
+     * @Route("/mon-comite-soutien/{slug}", name="app_citizen_project_committee_support")
+     * @Method("GET")
```

Comment:
This action should be `POST`. 
There is also a security issue and a CSRF token should be used to fix it.

Reply:
this URL is call on email send to committee supervisor

Reply:
Ok but we still need to fix the security issue.
I see 2 solutions : 
    - Create an intermediary page that would ask a confirmation to the user with a form (the simplest and best solution to me).
    - Use any kind of token (JWT, hmac time limited, a one time random secret, or whatever). Just like we do with password reset request.

Reply:
👍 to send a temporary token, as done with account creation confirmation.